app/containers/List/ListItem.tsx

```diff
@@ -91,7 +91,7 @@ const Content = React.memo(
 				<View style={styles.textContainer}>
 					<View style={styles.textAlertContainer}>
 						<Text style={[styles.title, { color: color || themes[theme].titleText }]} numberOfLines={1}>
-							{translateTitle ? I18n.t(title) : title}
+							{translateTitle ? I18n.t(title || '') : title}
```

Comment:
I've seen this on some other places on this PR.
If the code reaches `''`, it's actually going to show to the end user an awkward message [like this](https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/3960).
A better way to fix this warning is:

```suggestion
							{translateTitle && title ? I18n.t(title) : title}
```

Please review your PR and change it on other places as well.